File: app/src/main/java/xyz/jtmiles/beastforgw2/models/InfixUpgrade.java

package xyz.jtmiles.beastforgw2.models;

import java.util.ArrayList;
import java.util.List;

public class InfixUpgrade {

    private Integer id;
    private List<Attribute> attributes = new ArrayList<Attribute>();

    /**
     * 
     * @return
     *     The id
     */
    public Integer getId() {
        return id;
    }

    /**
     * 
     * @param id
     *     The id
     */
    public void setId(Integer id) {
        this.id = id;
    }

    /**
     * 
     * @return
     *     The attributes
     */
    public List<Attribute> getAttributes() {
        return attributes;
    }

    /**
     * 
     * @param attributes
     *     The attributes
     */
    public void setAttributes(List<Attribute> attributes) {
        this.attributes = attributes;
    }

}

File: app/src/main/java/xyz/jtmiles/beastforgw2/models/InfixUpgrade.kt
package xyz.jtmiles.beastforgw2.models

import java.io.Serializable

data class InfixUpgrade(val id: Int, val buff: Buff, val attributes: List<Attribute>) : Serializable {
    companion object {
        private val serialVersionUID = 0L
    }
}
